src/support/utf8.c

```diff
@@ -380,36 +368,6 @@ size_t u8_read_escape_sequence(const char *str, size_t ssz, uint32_t *dest)
     return i;
 }
 
-/* convert a string with literal \uxxxx or \Uxxxxxxxx characters to UTF-8
-   example: u8_unescape(mybuf, 256, "hello\\u220e")
-   note the double backslash is needed if called on a C string literal */
-size_t u8_unescape(char *buf, size_t sz, const char *src)
```

Comment:
Argh!  That's what happens when I code at 40,000 feet!  (the change was still in my editor, unsaved, and the compiler doesn't complain).
Fixed now, and I saw another function (and declaration!) to remove.